test-it/ExpectedResults/ComplexMathFunctions.vtr

```diff
@@ -40,3 +40,5 @@ Log (0 99) => (4.59512 1.5708)
 Log2 (0 99) => (6.62936 2.26618)
 Log10 (0 99) => (1.99564 0.682188)
 Log10 ((1.75517 0.958851) (1.62091 2.52441)) => ((0.30103 0.217147) (0.477121 0.434294))
+Absolute Double (-96.7324 -265.621) => 282.6865193998785912
```

Comment:
It might be flaky to expect a floating point number to this precision. May not make this test portable. I wonder whether we should do an epsilon comparison in .via against the expected value. Thoughts @cglzaguilar @spathiwa ?